mbedtls-sys/build/build.rs

```diff
@@ -65,6 +61,10 @@ impl BuildConfig {
                 if have_feature("custom_threading") {
                     writeln!(f, "typedef void* mbedtls_threading_mutex_t;")?;
                 }
+                if have_feature("custom_time") {
```

Comment:
Why is this suddenly needed?

Reply:
Discussed offline: The same issue existed earlier but was a warning. The `MBEDTLS_FATAL_WARNINGS` in `CMakeLists.txt` is new in mbedtls. This turns warnings into errors.